Sources/Benchmarks/Utils.swift

```diff
@@ -0,0 +1,55 @@
+import Foundation
+import MongoSwift
+
+let dataPath = "./data"
+
+struct TestFile {
+    let name: String
+    let size: Double
+    let json: String
+
+    init(name: String, size: Double) {
+        self.name = name
+        self.size = size
+        // we only call this method with known valid paths, so it won't fail.
+        // swiftlint:disable:next force_try
+        self.json = try! String(contentsOf: URL(fileURLWithPath: "\(dataPath)/\(name).json"))
+    }
+}
+
+/// Measure the time for a single execution of the provided closure.
+func measureTime(_ operation: () throws -> Void) throws -> Double {
```

Comment:
`TimeInterval` is just a typealias of `Double`, so I think we could just it everywhere in this file instead of `Double` to reduce some of the casting.